config/environments/development.rb

```diff
@@ -8,6 +8,8 @@
   # it changes. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
   config.enable_reloading = true
+  config.assets.debug = false
```

Comment:
It might help to turn it to true so you can get errors when the assets have problems